### PR TITLE
Shivani m

### PR DESCRIPTION
**Pull request by smaurya30**

### CODE REVIEW

**Review by Tirth-Patel903 — COMMENTED** [review]
1. There is lack of commenting because of that not able to find code's actual reason need to understand whole module.
2. I can Only able to run one code and other getting error of python don't know what error but related to path 
3. Code is incomplete till date there is missing some projects.
4. In UI task not good graphical user interface available Need to proper apart from message its looks like error in blue color.
5. result is not showed in right way it is more confusing.
6. Good note :
	- Try and catch block
	- designing of code is good 
	- Variable names are perfect only some are confusing
7. Test file has no comments variable names are not make sense 
8. Hard to understand code flow and logic it is too much complicated and also there is no comments available.

File: Shivani_Maurya/WordleLab10/src/python/HW_Shivani_Maurya_utility.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            # f = open("FiveLtterWord.txt", "w")
+            for word in words:
+                if len(word) == 5:
+                    f.write(word)
+                    f.write("\n")
+            f.close()
+        except:
+            print("Error:", sys.exc_info()[0], " in createFiveLetterWordsFilemethod, Utility module, occurred.".__str__())
+        # finally:        
+        #     f.close()
```
Comment (smohan98) [inline Shivani_Maurya/WordleLab10/src/python/HW_Shivani_Maurya_utility.py:40]: Would have been good coding practice if this code was not commented.

File: Shivani_Maurya/WordleLab10/src/python/HW03_Shivani_Maurya_wordle.py [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+        self.inputWordList = inputWordList
+        self.wordleWordList = wordleWordList
+        self.result = ['"', '"', '"', '"', '"']
+        self.resultString = ""
+        self.u = utility.Utility()
+        
+    def __str__(self):
+        return self
+    
+    def wordHelper(self, badWordSet, greenLetters, yellowLetters):
```
Comment (smohan98) [inline Shivani_Maurya/WordleLab10/src/python/HW03_Shivani_Maurya_wordle.py:39]: Could use comments to explain different functions

File: Shivani_Maurya/WordleLab10/src/python/HW03_Shivani_Maurya_wordle.py [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+        
+            fiveLetterWords = self.u.readWordRankFile()
+        
+            greenValCount = 0
+            for idxListVal in greenLetters.values():
+                greenValCount += len(idxListVal)
+            # yellowValCount = 0
+            # for idxListVal in yellowLetters.values():
+            #     yellowValCount += len(idxListVal)
+
```
Comment (smohan98) [inline Shivani_Maurya/WordleLab10/src/python/HW03_Shivani_Maurya_wordle.py:84]: Quite a few prior changes that are commented are not removed.

File: Shivani_Maurya/WordleLab10/src/python/HW03_Shivani_Maurya_ui.py [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+            return flag, winCount, Counter, gamesPlayed
+        except:
+            print("Error:", sys.exc_info()[0], " in wordleUi method, UI Module occurred.".__str__())
+    
+            
+    def statsFuncs(self):
+        try:
+            self.u.createFiveLetterWordsFile()
+            letterFreqDictRes = self.s.letterFreq()
+            convertLetterFreqDictRes = self.s.convertListToTuple(letterFreqDictRes.copy())
```
Comment (smohan98) [inline Shivani_Maurya/WordleLab10/src/python/HW03_Shivani_Maurya_ui.py:150]: Easy to understand variable names, made it easy to understand code

File: Shivani_Maurya/WordleLab10/src/python/HW03_Shivani_Maurya_dictionary.py [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+import logging
+
+'''
+Psuedocode:
+
+1. Read the file from resources folder
+2. put the dictionary wordws in a list
+3. Pick a random word for Wordle Game
+
+'''
```
Comment (smohan98) [inline Shivani_Maurya/WordleLab10/src/python/HW03_Shivani_Maurya_dictionary.py:14]: Good design patterns used.

**Review by smohan98 — APPROVED** [review]
Added code review comments.
The 2 Pycache folders and its respective files could be removed since those files are not required.

Thank you.

**Pull request merged**